packages/postgres-database/tests/test_delete_projects_and_users.py

```diff
@@ -84,7 +85,7 @@ async def test_insert_user(engine):
 
 async def test_count_users(engine):
     async with engine.acquire() as conn:
-        users_count = await conn.scalar(users.count())
+        users_count = await conn.scalar(sa.select([func.count()]).select_from(users))
```

Comment:
I see this change in many places. Is this a new way to count or was it simply wrong before?

Reply:
query.count was removed